### ISSUE TITLE
Warbler does not delete temporary files on Windows

### ISSUE THREAD

**Issue by mamantoha** [jruby/warbler#194]
We are using warbler to run an application, which is run on Windows and Linux.
After running jar, temporary files are not being deleted. We were able to reproduce the issue on Windows machine.
Biggest problem is that when running next time it creates another set of tmp files (does not override previous at least)

Eventually C:\Users\user\AppData\Local\Temp is clogged with `jruby<timestamp>extract` directories. Hopefully it's something easy to fix.
- warbler-1.4.0.dev
- jruby 1.7.4

Thank you in advance.
##

**Comment by ajf** [comment]
This isn't confined to windows -- we're seeing the same thing on Linux, with warbler 1.4.3 and jruby 1.7.10

**Comment by ryanrolland** [comment]
It would be nice if there was a mode to reuse the same temp exploded war directory. I can think of two benefits of this approach:
1) You avoid this issue, i.e. too much disk usage.
2) You could avoid the processor utilization to run the decompression on every invocation.

**Comment by sergio-fry** [comment]
👍 I have the same issue with 2.0.5

jruby-core, jruby-stdlib files are not recycled

**Comment by imweijh** [comment]
same issue here.
warble created jar on windows leaving lots of temporary files

**Comment by mohits** [comment]
Yes that's currently the case. You need a housekeeping job that cleans them up. You can recycle the path as linked further up in this issue.
